.github/workflows/ci.yaml

```diff
@@ -19,3 +22,33 @@ jobs:
         run: go get .
       - name: Test
         run: make test
+
+  release:
+    needs: test
+    runs-on: ubuntu-latest
+    if: ${{ github.ref_type == 'tag' }}
+    steps:
+      - uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Could you explain why we need `docker/login-action`?

Reply:
If we don't use it. The goreleaser uses v1 of it. Which gives error.

Reply:
I think it is not correct and I have already known another way without using `docker/login-action`.
I will do it by myself, but thanks for your pr.